microsoft-edge/devtools-guide-chromium/evaluate-performance/reference.md

```diff
@@ -374,56 +398,46 @@ Click an interaction to view more information about it in the **Summary** tab.
 <!-- ------------------------------ -->
 #### Analyze frames per second (FPS)
 
-DevTools provides numerous ways to analyze frames per second:
+DevTools provides two ways to analyze frames per second:
 
-*  Use [the FPS chart](#the-fps-chart) to get an overview of FPS over the duration of the recording.
 *  Use [the Frames section](#the-frames-section) to view how long a particular frame took.
 *  Use the **FPS meter** for a realtime estimate of FPS as the page runs.  See [View frames per second in realtime with the FPS meter](#view-frames-per-second-in-realtime-with-the-fps-meter).
 
 
-###### The FPS chart
```

Comment:
This got removed from DevTools, see https://bugs.chromium.org/p/chromium/issues/detail?id=1505063

Reply:
"fps chart" no longer is in article